src/icalendar/tests/test_issue_500_vboolean_for_parameter.py

```diff
@@ -0,0 +1,9 @@
+from icalendar import Event, vBoolean, vCalAddress
+
+def test_vBoolean_can_be_used_as_parameter_issue_500(events):
```

Comment:
Should there be tests for some of the other types or is that excessive?  Basically everything with an `.to_ical()` method works and I'm assuming there's test already for those methods.

Reply:
There are tests for [proper encoding](https://github.com/collective/icalendar/blob/master/src/icalendar/tests/test_unit_prop.py) for other v types, but I don't think there are any for checking if they can be used as a param. If you've got the time to file a PR with a parametrization of this test for other v types I'll gladly accept it :) if you don't, I will try to update the test sometime next week :)